client/state/sites/hooks/with-sites-sorting.tsx

```diff
@@ -2,10 +2,12 @@ import { createHigherOrderComponent } from '@wordpress/compose';
 import { ComponentType } from 'react';
 import { useSitesSorting } from './use-sites-sorting';
 
+export type SitesSortingPreferenceProps = ReturnType< typeof useSitesSorting >;
```

Comment:
`useSitesSorting` will be renamed to `useSitesSortingPreference` in a follow-up PR to maintain consistency.